### DISCUSSION TITLE
DatePicker validation message

### DISCUSSION THREAD

**Discussion by giladvidal** [UI5/webcomponents-react#3003] [Q&A] [answered]
Hi,
I'm using this component in a form with schema validation. When entering invalid value I'd like the user to see this message below the component (the way it is now) but when submitting empty, I'd like to only see the red border.

![image](https://user-images.githubusercontent.com/56625024/170955093-350549b2-49bb-46e7-ac0a-362ffc3a8490.png)

Also, is it possible to control the focus after schema validation? it looks like the last field to be validated is where the focus remains.

Thanks,
Gilad

**Comment by Lukas742** [comment, marked as answer]
Hi @giladvidal 

I don't think it's currently possible to disable the default error message, you can only overwrite it with something else. If you feel like this should be possible please create a feature request in the [UI5 Web Components repo](https://github.com/SAP/ui5-webcomponents/issues/new/choose).  

To control the focus you can use the HTMLElement `focus()` method or the `focus()` [method of the web component](https://github.com/SAP/ui5-webcomponents-react/blob/cbf79a54edc9013d5f36a143b0c8e2d4e57fdac8/packages/main/src/interfaces/Ui5DomRef.ts#L32-L35). In gerneral they are basically doing the same - the custom method only makes sure that the correct element is marked as activeElement.
